frontend/scripts/pages/profile-place.page.jsx

```diff
@@ -160,156 +160,47 @@ const _build = (data, { year, showMiniSankey }, store) => {
     document.querySelector('.deforestation').classList.toggle('is-hidden', true);
   }
 
-  if (showMiniSankey === 'true') {
-    document.querySelectorAll('.mini-sankey-container')
-      .forEach((el) => {
-        el.classList.toggle('is-hidden', false);
-      });
-
-    // query: nodeId, targetColumnId + commodity and year
-    const tradersSankeyData = {
-      name: 'QUERÊNCIA',
-      targetNodes: [{
-        id: 2,
-        name: 'Other',
-        isAggregated: true,
-        height: 0.2
-        // isDomesticConsumption: null,
-      }, {
-        id: 588,
-        name: 'Cargill',
-        height: 0.4
-      }, {
-        id: 588,
-        name: 'Hello, I have such a long name I need 3 lines',
-        height: 0.2
-      }, {
-        id: 588,
-        name: 'Sometimes, you gotta have more lines',
-        height: 0.1
-      }, {
-        id: 588,
-        name: 'Cargill',
-        height: 0.029
-      }, {
-        id: 588,
-        name: 'Cargill',
-        height: 0.021
-      }, {
-        id: 588,
-        name: 'Im very long but sadly the node hieght is too small',
-        height: 0.05
-      }]
-    };
-
-    // query: nodeId, targetColumnId + commodity and year
-    const consumersSankeyData = {
-      name: 'QUERÊNCIA',
-      targetNodes: [{
-        name: 'Others',
-        isAggregated: true,
-        height: 0.2
-      }, {
-        name: 'China',
-        height: 0.4
-      }, {
-        name: 'Brazil',
-        height: 0.2
-      }, {
-        name: 'Germany',
-        height: 0.1
-      }, {
-        name: 'South Korea',
-        height: 0.029
-      }, {
-        name: 'Thailand',
-        height: 0.021
-      }, {
-        name: 'Spain',
-        height: 0.05
-      }]
-    };
-
-    render(
-      <MiniSankey
-        data={tradersSankeyData}
-        targetLink="actor"
-      />,
-      document.getElementById('js-traders-sankey')
-    );
+  document.querySelectorAll('.mini-sankey-container')
```

Comment:
does this work? 👀

Reply:
seems to, but it's actually not needed, so I'll remove it

Reply:
ok I'm impressed I didn't know this. However, for the future => https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach

It's not supported on IE11 :(